crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/WireCoreCryptoUniffi.h

```diff
@@ -0,0 +1,27 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for WireCoreCryptoUniffi.
+FOUNDATION_EXPORT double WireCoreCryptoUniffiVersionNumber;
+
+//! Project version string for WireCoreCryptoUniffi.
+FOUNDATION_EXPORT const unsigned char WireCoreCryptoUniffiVersionString[];
```

Comment:
Why do we need these two symbols? Is this being used by something outside this module?

Reply:
This code is generated when creating a new ios framework.